packages/mobile/src/components/composer-input/ComposerInput.tsx

```diff
@@ -28,7 +36,9 @@ import LoadingSpinner from '../loading-spinner/LoadingSpinner'
 
 import type { ComposerInputProps } from './types'
 
-const BACKSPACE_KEY = 'Backspace'
+// const BACKSPACE_KEY = 'Backspace'
```

Comment:
Can delete this now

Reply:
added this back in the backspace handling work coming up